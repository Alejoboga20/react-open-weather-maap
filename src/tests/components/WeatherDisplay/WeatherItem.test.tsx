import { shallow } from 'enzyme';
import {
  WeatherItem,
  WeatherItemProps,
} from '../../../components/WeatherDisplay/WeatherItem';

const weatherItemProps: WeatherItemProps = {
  temp: 100,
};

describe('WeatherItem Tests', () => {
  test('should match Snapshot', () => {
    const wrapper = shallow(<WeatherItem {...weatherItemProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
