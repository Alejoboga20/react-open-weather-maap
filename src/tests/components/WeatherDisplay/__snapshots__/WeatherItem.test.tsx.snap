// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherItem Tests should match Snapshot 1`] = `
<div
  className="weatherItem_container"
>
  <p>
    100
     Â°
  </p>
</div>
`;
